docker/docker-compose.yaml

```
@@ -27,9 +25,20 @@ services:
       - "${BITCOINCASH_ADDRESS}"
 
   ##
-  ## Dogecoin
+  ## DigiByte
   ##
+  digibyte:
+    build:
+      context: ./digibyte
+    ports:
+      - "0.0.0.0:18443:18443"
```

These ports are already reserved by Bitcoin. Consider changing to:
```suggestion
      - "0.0.0.0:20443:18443"
```